app/routes/game.js

```diff
@@ -48,6 +49,21 @@ export default Ember.Route.extend({
 
       // Save the model with the new participant state.
       return model.save();
+    })
+    .then((game) => {
+      //load all the real card models into the store for future use
+      var players = game.get('players');
+      var promises = players.reduce((prev, p) => {
+        prev = prev.concat(p.get('gameCards').map((card) => store.find('card', card.cardId)));
```

Comment:
When creating a new game, this `p.get` was not a function at times. I used the following workaround, but not sure if this is the best approach.

```javascript
var gameCards = Ember.get(p, 'gameCards') || [];
```